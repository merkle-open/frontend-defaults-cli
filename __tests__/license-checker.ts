/// <reference types="@types/jest" />
import { apiIt } from './shared';

describe('license-checker', () => {
	it('default', async () =>
		await apiIt('license-checker', {
			licenseChecker: true,
		}));
});
