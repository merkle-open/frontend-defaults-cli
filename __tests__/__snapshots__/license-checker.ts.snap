// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`license-checker default 1`] = `
<div
  dangerouslySetInnerHTML={
    Object {
      "__html": "{
  \\"scripts\\": {
    \\"lint\\": \\"npm-run-all lint:*\\",
    \\"lint:license\\": \\"license-checker --production --summary --exclude \\\\\\"Apache-2.0, BSD, ISC, LGPL, MIT, MPL\\\\\\" --failOn \\\\\\"AGPL; EPL; GPL\\\\\\"\\"
  },
  \\"devDependencies\\": {
    \\"license-checker\\": \\"25.0.1\\",
    \\"npm-run-all\\": \\"4.1.5\\"
  }
}",
    }
  }
/>
`;

exports[`license-checker default 2`] = `
<div
  dangerouslySetInnerHTML={
    Object {
      "__html": "package.json",
    }
  }
/>
`;
